master/propidx.html

```diff
@@ -96,6 +96,16 @@ <h1>Property Index</h1>
           <td>yes</td>
           <td>as specified</td>
         </tr>
+        <tr>
+          <th><a>'d'</a></th>
+          <td>none | path(<a>&lt;string&gt;</a>)</td>
+          <td>none</td>
+          <td><a>'path'</a> elements</td>
+          <td>no</td>
+          <td>N/A</td>
+          <td><a href="https://www.w3.org/TR/2008/REC-CSS2-20080411/media.html#visual-media-group">visual</a></td>
+          <td>yes</td>
```

Comment:
We've updated the Property Index to specify the type of animation, not just yes/no. There's also a "computed value" column. Which makes me realize that we never resolved #321 on what that should be (do you normalize before computed/used value?).  For now:

```suggestion
          <td>as path data string</td>
          <td>as specified</td>
```

Ideally, "as path data string" would be linked to the definition of how animation works.